ingress-nginx/camunda-values.yaml

```diff
@@ -47,6 +49,24 @@ connectors:
 #    - name: CAMUNDA_OPERATE_CLIENT_PASSWORD
 #      value: demo
 
+webModeler:
+  enabled: true
```

Comment:
It's great to have the ingress configuration for the Web Modeler but I'd disable it by default.